sidebars.js

```diff
@@ -31,7 +35,8 @@ const sidebars = {
       items: [
         "tutorials/vs-code-docker", 
         "tutorials/vs-code-remote-containers", 
-        "tutorials/rancher-ui-on-rancher-desktop"
+        "tutorials/rancher-ui-on-rancher-desktop",
+        "tutorials/hello-world-example",
```

Comment:
Better to move hello-world-page as the first item under `Tutorial`

Reply:
Updated as suggested.